release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -17,7 +17,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 
 //Red Hat did not publicly release {product-title} 4.10.0 as the GA version and, instead, is releasing {product-title} 4.10.TBD as the GA version.
 
-{product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {console-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
+{product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {console-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premises or cloud environments.
```

Comment:
on-premise is actually correct: https://redhat-documentation.github.io/supplementary-style-guide/#on-premise